.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  # Enable version updates for Docker
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

We would definitely like dependabot to help us with python dependencies. We might as well try it for npm packages too (we can tweak this later if needed)

```suggestion
      interval: "weekly"
      
  - package-ecosystem: "pip"
    versioning-strategy: "increase-if-necessary"
    directory: "/"
    schedule:
        interval: "weekly"
  - package-ecosystem: "npm"
    versioning-strategy: "widen"
    directory: "/"
    schedule:
        interval: "weekly"
```